.circleci/config.yml

```diff
@@ -1,8 +1,128 @@
 version: 2
 jobs:
-  test:
+  test_min:
+    docker:
+      - image: node:6.10
+      - image: rabbitmq:3.4
+
+    steps:
+      - checkout
+      - restore_cache:
+          key: dependency-cache-{{ .Branch }}-{{ checksum "package.json" }}
+      - run:
+          name: Install node modules
+          command: npm i
+      - save_cache:
+          key: dependency-cache-{{ .Branch }}-{{ checksum "package.json" }}
+          paths:
+            - node_modules
+            - /root/.npm
+      - run:
+          name: Run tests
+          command: npm test
+      - run:
+          name: Codecov
+          command: bash <(curl -s https://codecov.io/bash)
+
+  test_v6:
```

Comment:
Is this one useful since there is already the job with node 6.10?

Reply:
I think 6.10 is not the latest node 6 (6.11), so it's still useful IMHO ...